requirements/base.txt

```diff
@@ -34,8 +34,8 @@ edx-enterprise-data==2.1.0  # via -r requirements/base.in
 edx-opaque-keys==2.1.0    # via -r requirements/base.in, edx-ccx-keys, edx-drf-extensions, edx-enterprise-data
 edx-rbac==1.2.1           # via edx-enterprise-data
 edx-rest-api-client==5.2.1  # via -r requirements/base.in, edx-enterprise-data
-elasticsearch-dsl==0.0.11  # via -c requirements/constraints.txt, -r requirements/base.in
-elasticsearch==1.9.0      # via elasticsearch-dsl
+elasticsearch-dsl==7.2.1  # via -c requirements/constraints.txt, -r requirements/base.in
```

Comment:
Seryozha, this is a project for edx and unfortunately, we cannot do it as we do for our raccoon projects. You cannot touch .txt files to make dependencies. These files are generated from .in files. It is safe and reliable. The algorithm should be as follows. We change the package version in the .in file and compile it to get a .txt file. After we commit both .in and .txt.

Reply:
Done

Reply:
I cannot find changes related to the ES libs in the .in files. While we use ES 7.x.x our libs should be limited to `<8.0.0` in the .in files

Reply:
ES lib version defines and gets to update from the  `constraints.txt` file. In this file sets only `elasticsearch-dsl` lib